src/AssetsDetectionController.ts

```diff
@@ -195,16 +204,28 @@ export class AssetsDetectionController extends BaseController<AssetsDetectionCon
 				description,
 				asset_contract: { address }
 			} = collectible;
-			await assetsController.addCollectible(
-				address,
-				Number(token_id),
-				{
-					description,
-					image: image_preview_url,
-					name
-				},
-				true
-			);
+
+			let ignored = null;
+			/* istanbul ignore else */
+			if (ignoredCollectibles && ignoredCollectibles.length) {
```

Comment:
after our discussion `ignoredCollectibles` should be always at least an empty array